src/screens/Login.tsx

```diff
@@ -11,10 +14,21 @@ export function Login() {
     event.preventDefault();
 
     try {
```

Comment:
You can move the try-catch block to the service to make this file cleaner. Consider adding a `success` field to the value returned by the service.